lib/core/src/lib/comments/comment-list/comment-list.component.html

```diff
@@ -3,10 +3,12 @@
                   (click)="selectComment(comment)"
                   class="adf-comment-list-item">
         <div class="adf-comment-img-container">
-            <div *ngIf="!comment.hasAvatarPicture" class="adf-comment-user-icon">{{ comment.userInitials }}</div>
-            <img *ngIf="comment.hasAvatarPicture" class="adf-people-img"
-                 [alt]="'COMMENTS.PROFILE_IMAGE' | translate"
-                 [src]="getUserImage(comment.createdBy.id.toString())" />
+            <div *ngIf="!getUserImage(comment.createdBy)" class="adf-comment-user-icon">
```

Comment:
We are in Angular 19 now so we can use new control flow and variables in html to avoid calling getUserImage multiple times. Something like that: 
```html
@if (getUserImage(comment.createdBy); as userImage) {
  <img class="adf-people-img"
           [alt]="'COMMENTS.PROFILE_ALT' | translate:{ name: comment.userDisplayName }"
           [src]="userImage" />
} @else {
  <div class="adf-comment-user-icon">
    {{ comment.userInitials }}
  </div>
}
```

Thanks to else we don't need to repeat ngIf in both places. I added also "as userImage" in if to avoid calculating the same when assigning to src